batch/batch/front_end/templates/batches.html

```diff
@@ -65,12 +68,23 @@ <h1>Batches</h1>
 	      {{ batch['attributes']['name'] }}
 	      {% endif %}
 	    </td>
+	    <td>{{ batch['time_created'] }}</td>
```

Comment:
It's confusing that created and completed are displayed but the duration is the time between an undisplayed value, closed, and complete. Should we display closed instead? Created is more like "Queued" or something, right?